odl/test/trafos/fourier_test.py

```diff
@@ -631,6 +631,53 @@ def char_rect(x):
             assert all_almost_equal(ft.adjoint(ft(char_rect)), discr_rect)
 
 
+def _test_adjoint(operator):
+    """Test adjoint of operator satisfies the definition."""
+    x = noise_element(operator.domain)
+    y = noise_element(operator.range)
```

Comment:
These are expected to fail. The adjoint is set equal to the inverse, not (necessarily) the adjoint of the discretized operator. So for functions that fill the entire space, FT is expected to be produce errors. The inverse tests would probably also fail for these cases.
But perhaps the choice of setting adjoint = inverse needs a revisit.